runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt

```diff
@@ -56,6 +56,21 @@ fun main(args: Array<String>) {
         }
     }
 
+    val profilerName = config.advanced["jvmProfiler"]
+    when (profilerName) {
+        "gc" -> jmhOptions.addProfiler("gc")
+        "stack" -> jmhOptions.addProfiler("stack")
+        "cl" -> jmhOptions.addProfiler("cl")
+        "comp" -> jmhOptions.addProfiler("comp")
+        "perf" -> jmhOptions.addProfiler("perf")
+        "perfnorm" -> jmhOptions.addProfiler("perfnorm")
+        "perfasm" -> jmhOptions.addProfiler("perfasm")
+        "xperfasm" -> jmhOptions.addProfiler("xperfasm")
+        "dtraceasm" -> jmhOptions.addProfiler("dtraceasm")
+        null -> {}
+        else -> throw IllegalArgumentException("Invalid value for 'jvmProfiler': $profilerName. Accepted values: gc, stack, cl, comp")
+    }
+
     val reportFormat = ResultFormatType.valueOf(config.reportFormat.uppercase())
     val reporter = BenchmarkProgress.create(config.traceFormat)
     val output = JmhOutputFormat(reporter, config.name)
```

Comment:
When a profiler (e.g., `gc`) is added, each iteration result includes not just the primary result but also a secondary one. Currently, our reporters seem to ignore these secondary results from the iterations. To observe the discrepancy, one can run a benchmark directly with JMH adding a profiler, and compare it to running a benchmark via the kx-benchmark plugin, also with a profiler added.

Could you please ensure that the secondary result (if not empty) of each iteration is reported as well?
The secondary result of a set of iterations should also be reported.